enginetest/script_queries.go

```diff
@@ -456,6 +456,52 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "Test cases on select into statement",
+		SetUpScript: []string{
+			"SELECT 1 INTO @abc",
+			"SELECT * FROM (VALUES ROW(22,44,88)) AS t INTO @x,@y,@z",
+			"CREATE TABLE tab1 (id int primary key, v1 int)",
+			"INSERT INTO tab1 VALUES (1, 1), (2, 3), (3, 6)",
+			"SELECT id FROM tab1 ORDER BY id DESC LIMIT 1 INTO @myVar",
```

Comment:
It would be good to have an example us `into <variable>` where into is not in the last position. I know we have tests covering that at the grammar layer, but one quick sanity test for that at this layer would still be helpful.

Reply:
On a similar line... I'd also consider a couple of tests with into that generate errors (in addition to the dumpfile/outfile tests below).  For example... using INTO twice would be a good one to sanity check that our error handling code at the parser layer works correctly here. 

Good job testing the new `ErrMoreThanOneRow` error kind. How about testing the error case where we try to select one row with 3 columns, but only supply 2 vars to select those results into?